ports/esp32/boards/GENERIC_S2/mpconfigboard.h

```diff
@@ -1,8 +1,5 @@
-#define MICROPY_HW_BOARD_NAME               "ESP32S2 module"
-#define MICROPY_HW_MCU_NAME                 "ESP32S2"
+#define MICROPY_HW_BOARD_NAME "Generic ESP32S2 module"
+#define MICROPY_HW_MCU_NAME "ESP32S2"
 
-#define MICROPY_PY_BLUETOOTH                (0)
-#define MICROPY_HW_ENABLE_SDCARD            (0)
-
-// Enable UART REPL for modules that have an external USB-UART and don't use native USB.
-#define MICROPY_HW_ENABLE_UART_REPL         (1)
```

Comment:
I see you removed this... what was the reason?

Related to this, it seems that `CONFIG_USB_AND_UART` is not a real thing, at least I could not find it in the IDF or TinyUSB component...

Reply:
Yep, that should be still there. Updated

Also removed `CONFIG_USB_AND_UART` from all S2 boards.